h2/src/main/org/h2/command/Parser.java

```diff
@@ -2631,6 +2631,17 @@ private Expression readFactor() {
         }
     }
 
+    private Expression readFilterCondition() {
```

Comment:
when you add methods like this, please put them below the call site, since Parser is largely ordered in a top-down approach